tests/minifier.js

```diff
@@ -535,34 +535,34 @@ test('remove CDATA sections from scripts/styles', function() {
   output = '<script>alert(11)</script>';
   equal(minify(input, { minifyJS: true }), output);
 
-  input = '<style><![CDATA[\np.a{backgourd:red}\n]]></style>';
```

Comment:
If it weren't for VS Code suggesting keywords harvested from the file, I wouldn't have spotted this :disappointed_relieved: